doc/sphinx/language/core/records.rst

```diff
@@ -10,6 +10,9 @@ components, called :gdef:`fields <field>`, can be accessed with
 mathematical structures, such as groups or rings, hence the
 synonym :cmd:`Structure`.
 
+Defining record types
+~~~~~~~~~~~~~~~~~~~~~
+
 .. _record_grammar:
 
 .. cmd:: {| Record | Structure } @record_definition
```

Comment:
I just noticed that `Record` actually support the `with` clause to simultaneously define several (non-recursive) records. @herbelin If we stop documenting this form, then we should also deprecate it.

Reply:
It doesn't work.  IIRC @herbelin removed that recently (though when I look at the grammar, it looks like it should work-maybe a case where the parser can't fully handle the grammar as written).

![image](https://user-images.githubusercontent.com/1253341/154161650-3a2ffd53-ce4a-4438-b414-c39d81bb84c9.png)

Reply:
Yes, the `with` was removed in #15484.

Reply:
Oh right, sorry I was not testing an up-to-date version.